packages/expo-web-browser/android/src/main/java/expo/modules/webbrowser/CustomTabsConnectionHelper.java

```diff
@@ -1,115 +1,13 @@
 package expo.modules.webbrowser;
```

Comment:
As we're aiming for Kotlin supremacy, I'll convert this to kotlin 😉

Reply:
Same as below :)